crates/criterion_compat/criterion-0.5.1/Cargo.toml

```diff
@@ -152,7 +156,7 @@ default-features = false
 version = "0.8"
```

Comment:
let's change `default-features` to `default_features` in this file to avoid this pesky warning

```
warning: /home/guillaume/codspeed/rust/crates/criterion_compat/criterion-0.5.1/Cargo.toml: `default_features` is deprecated in favor of `default-features` and will not work in the 2024 edition
(in the `futures` dependency)
```